### PR TITLE
add kafka extension

### PR DESCRIPTION
**Pull request by itscaro**
Test: rdkafka

### CODE REVIEW

**Comment by mlocati** [comment]
Please follow [there instructions](https://github.com/mlocati/docker-php-extension-installer#adding-support-to-a-new-php-extension)

File: install-php-extensions [hunk 1]
```diff
@@ -893,6 +901,11 @@ installPECLModule () {
 				installPECLModule_actual="$2-2.0.10"
 			fi
 			;;
+		rdkafka)
+			if test ! -z "$(apt-cache show librdkafka++1 | grep -E '^Version: 0.9')"; then
```
Comment (mlocati) [inline install-php-extensions]: Will this work with Alpine?

Comment (itscaro) [reply install-php-extensions]: thanks

File: data/supported-extensions [hunk 2]
```diff
@@ -42,6 +42,7 @@ pgsql 5.6 7.0 7.1 7.2 7.3 7.4
 protobuf 5.6 7.0 7.1 7.2 7.3 7.4
 pspell 5.6 7.0 7.1 7.2 7.3 7.4
 pthreads 5.6 7.0
+rdkafka 5.6 7.0 7.1 7.2 7.3 7.4
```
Comment (itscaro) [inline data/supported-extensions]: failed on alpine`, any idea?

```docker run --rm -it php:5.6-cli-alpine sh
/ # curl -o /usr/local/bin/install-php-extensions https://raw.githubusercontent.com/itscaro/docker-php-e
xtension-installer/kafka/install-php-extensions && chmod +x /usr/local/bin/install-php-extensions; insta
ll-php-extensions rdkafka
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 40968  100 40968    0     0   754k      0 --:--:-- --:--:-- --:--:--  754k
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
v3.8.4-250-g4fce639e9b [http://dl-cdn.alpinelinux.org/alpine/v3.8/main]
v3.8.4-244-g50796da3f3 [http://dl-cdn.alpinelinux.org/alpine/v3.8/community]
OK: 9577 distinct packages available
### INSTALLING REQUIRED PACKAGES ###
(1/5) Installing libgcc (6.4.0-r9)
(2/5) Installing libstdc++ (6.4.0-r9)
(3/5) Installing librdkafka (0.11.4-r1)
(4/5) Installing pkgconf (1.5.3-r0)
(5/5) Installing librdkafka-dev (0.11.4-r1)
Executing busybox-1.28.4-r3.trigger
OK: 26 MiB in 34 packages
### INSTALLING PECL MODULE rdkafka ###
downloading rdkafka-4.0.2.tgz ...
Starting to download rdkafka-4.0.2.tgz (39,349 bytes)
..........done: 39,349 bytes
32 source files, building
running: phpize
Configuring for:
PHP Api Version:         20131106
Zend Module Api No:      20131226
Zend Extension Api No:   220131226
Cannot find autoconf. Please check your autoconf installation and the
$PHP_AUTOCONF environment variable. Then, rerun this script.

ERROR: `phpize' failed```

Comment (mlocati) [reply data/supported-extensions]: You need to add `$PHPIZE_DEPS` to `buildRequiredPackageLists_volatile` (see other extensions for Alpine)

Comment (itscaro) [reply data/supported-extensions]: thanks, tested with 5.6-cli-alpine & 7.0-cli-alpine and it works :)

**Comment by mlocati** [comment]
:+1: thanks!

**Pull request merged**